plugin/evm/customethclient/example_test.go

```diff
@@ -0,0 +1,30 @@
+package customethclient
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/ava-labs/coreth/plugin/evm/customtypes"
+)
+
+const FujiAPIURI = "https://api.avax-test.network"
+
+func ExampleClient() {
+	ethC, err := Dial(FujiAPIURI + "/ext/bc/C/rpc")
+	if err != nil {
+		panic(err)
+	}
+	bc, err := ethC.BlockByNumber(context.Background(), nil)
```

Comment:
Maybe let's target a particular block number and convert this to a test with assertions? If it's the same block, the results should remain the same I think

Reply:
I'm not sure about adding a UT that hits external server (api.avax). I've looked for a possible mock server but could not really find one. Given that the original ethclient does not also have any test, I think this is fine. The intention here is not to test/show how client is being called, but rather how the extra types can be extracted.